src/modules/kvs/kvs.c

```diff
@@ -2703,6 +2727,30 @@ static void setroot_unpause_request_cb (flux_t *h, flux_msg_handler_t *mh,
         flux_log_error (h, "%s: flux_respond_error", __FUNCTION__);
 }
 
+static void reload_cb (flux_t *h,
```

Comment:
maybe naming this `config_reload_cb()` would be better in such a large code base as the kvs?